app/Http/Controllers/Api/TimeLog/ClockInDomain/Services/ClockInService.php

```diff
@@ -17,16 +17,16 @@ public function __construct(ClockInLogicContract $clockInLogicUtility)
         $this->clockInLogicUtility = $clockInLogicUtility;
     }
 
-    public function clockIn(string $timeStamp): array
+    public function clockIn(string $date, string $time): array
     {
         $user = Auth::user();
         $userId = $user->id;
 
         $this->clockInLogicUtility->verifyUserHasNotYetTimeLogged($userId);
```

Comment:
I feel like we might need to refactor verifyUserHasNotYetTimeLogged. You're checking if there's any record in the table where clock out is null, but should we check it only for the date theyre trying to clock in for?

Reply:
True, I'll adjust that. Should we have a null check as a daily task process. If they any user forgot to clock out, notify them?

Reply:
Or should we do that at the beginning of opening the app.